src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__container calendar__container--start-day-fri calendar__container--month-length-29 calendar__container--top">
+        <div class="calendar__day day-1"></div>
```

Comment:
```suggestion
        <div class="calendar__day"></div>
```
Remove this class everywhere. use nth-child instead

Reply:
Done